app/src/main/java/com/humberto/tasky/agenda/presentation/agenda_details/AgendaDetailsScreen.kt

```diff
@@ -114,12 +167,141 @@ private fun AgendaDetailsScreen(
             )
         }
     ) {
-        Column {
-            Text(
-                text = agendaItemType,
-                style = MaterialTheme.typography.headlineLarge,
-                color = MaterialTheme.colorScheme.onSurface
+        Column(
+            modifier = Modifier
+                .fillMaxSize()
+                .verticalScroll(rememberScrollState())
+        ) {
+            Column(
+                modifier = Modifier
+                    .padding(horizontal = 16.dp)
+                    .padding(top = 32.dp)
+            ) {
+                AgendaItemIndicator(agendaItemType = agendaItem.agendaItemType)
+                TaskyEditableField(
+                    isEditing = state.isEditing,
+                    modifier = Modifier
+                        .padding(top = 8.dp)
+                ) {
+                    Row(
+                        verticalAlignment = Alignment.CenterVertically
+                    ) {
+                        TaskyRadioButton(enabled = false)
+                        Spacer(modifier = Modifier.width(12.dp))
+                        Text(
+                            text = agendaItem.title
+                                .ifEmpty { stringResource(id = R.string.add_title) },
+                            style = MaterialTheme.typography.headlineMedium,
+                            color = if(agendaItem.title.isNotEmpty()) MaterialTheme.colorScheme.onSurface
+                                else TaskyGray
+                        )
+                    }
+                }
+                HorizontalDivider(
+                    color = MaterialTheme.colorScheme.tertiary
+                )
+                TaskyEditableField(
+                    isEditing = state.isEditing
+                ) {
+                    Text(
+                        text = agendaItem.description
+                            .ifEmpty { stringResource(id = R.string.add_description) },
+                        style = MaterialTheme.typography.labelMedium,
+                        color = if(agendaItem.description.isNotEmpty()) MaterialTheme.colorScheme.onSurface
+                            else TaskyGray,
+                        maxLines = 5,
+                        overflow = TextOverflow.Ellipsis
+                    )
+                }
+            }
+            PhotosSection(
+                agendaItemType = agendaItem.agendaItemType,
+                photos = agendaItem.photosUrlList
             )
```

Comment:
Given that the photos section should only be shown for events, I feel like passing the type and leaving the decision up to the composable is not the right design (because technically, each item will still have the `PhotosSection` node in its UI tree).
Shouldn't it rather be the responsibility of the screen to decide which composable is shown when? If I wouldn't know that this is event-only, I might be confused why adding this composable doesn't show anything for tasks and reminders.